dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/common/GridHeaders.java

```diff
@@ -45,6 +45,7 @@
 {
     private GridHeaders()
```

Comment:
we can remove this and annotate `GridHeaders` class with `@NoArgsConstructor(PRIVATE)`

Reply:
Done